ts/features/wallet/bancomat/screen/BancomatInformation.tsx

```diff
@@ -26,50 +27,48 @@ const styles = StyleSheet.create({
     width: "100%",
     borderColor: IOColors.blue
   },
-  brandLogo: {
-    width: 40,
-    height: 25,
-    resizeMode: "contain"
-  },
-  vPay: {
-    width: 25,
-    height: 25,
-    resizeMode: "contain"
-  },
-  row: {
-    flexDirection: "row"
+  titleContainer: {
+    flex: 1,
+    flexDirection: "row",
+    justifyContent: "space-between"
   }
 });
 
 /**
- * Display generic information on bancomat and a cta to start the onboarding of a new
+ * Display the logo bar and a cta to start the onboarding of a new
  * payment method.
- * TODO: this will be also visualized inside a bottomsheet after an addition of a new bancomat
  * @constructor
  */
 
-const BancomatInformation: React.FunctionComponent<Props> = props => (
-  <View>
-    {!props.hideCobrandTitle && (
-      <H4>{I18n.t("wallet.bancomat.details.debit.title")}</H4>
-    )}
-    <View spacer={true} />
-    <InternationalCircuitIconsBar />
-    <View spacer={true} />
-    <Markdown>{I18n.t("wallet.bancomat.details.debit.body")}</Markdown>
-    <View spacer={true} />
-    <ButtonDefaultOpacity
-      style={styles.button}
-      bordered={true}
-      onPress={() => {
-        props.onAddPaymentMethod?.();
-      }}
-      onPressWithGestureHandler={true}
-    >
-      <Label>{I18n.t("wallet.bancomat.details.debit.addCta")}</Label>
-    </ButtonDefaultOpacity>
-  </View>
-);
+const BancomatInformation: React.FunctionComponent<Props> = props => {
+  const { present } = BancomatInformationBottomSheet(props.onAddPaymentMethod);
```

Comment:
```suggestion
  const { present } = bancomatInformationBottomSheet(props.onAddPaymentMethod);
```

To keep consistency in the codebase, we use `PascalCase` for components and types, `camelCase` for functions and const.